### DISCUSSION TITLE
Being assigned a traitor mission that spawns items while your inventory is full drops aforementioned items on the ground. or equips them

### DISCUSSION THREAD

**Discussion by ghost** [FakeFishGames/Barotrauma#13052] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I was selected as traitor, given the sootman mission, and since my inventory was full my radio jammer and sootman ID dropped, and the bandana was equipped automatically.

### Reproduction steps

1. Roll traitor
2. Roll the sootman mission with a full inventory
3. Your very obvious traitor items will drop on the floor and you will be potentially outed as a traitor before you're even aware you're one.

### Bug prevalence

Just once

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.1.18.1 (Treacherous Tides)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Hmm, this is can definitely be a problem, but I'm not entirely sure how it could be addressed. Where else could the items be put if the inventory is full?

**Comment by ghost** [comment]
Perhaps hide it in a vent somewhere akin to old traitor missions? Maybe reduce the number of items spawned? The mission I encountered this on spawned a seperatist bandana which could be considered unnecessary since players could source their own face coverings.
